atom/browser/native_window_views.h

```diff
@@ -228,6 +230,9 @@ class NativeWindowViews : public NativeWindow,
   // In charge of running taskbar related APIs.
   TaskbarHost taskbar_host_;
 
+  // Memoized version of a11y check
+  bool checked_for_a11y_support_;
```

Comment:
This should be initialized to `false` in the constructor like `enabled_a11y_support_` is in [`atom/browser/native_window_views.cc`](https://github.com/electron/electron/blob/988e2334f59d8aaccdec394a9d9fa0c39920068d/atom/browser/native_window_views.cc#L141)